src/cba-list-new/shadow.css

```diff
@@ -202,9 +201,133 @@ li .hasInfo::after
   margin-bottom: 1px;
   background: no-repeat var(--icon-project);
 }
+span.row
+{
+  height: 20px;
+}
+
+#group
+{
+  padding-left: 8px;
+  padding-right: 22px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  position: relative;
+  height: 21px;
```

Comment:
Note: Why having fixed height?

Reply:
There is an issue with height. Somehow the hight of span is allways the same 39 px, height: auto or height% doesn't work, don't know yet why. Can we adjust the height of span in em ?